python-libraries/data-platform-catalogue/data_platform_catalogue/client.py

```diff
@@ -94,46 +121,66 @@ def create_or_update_database_service(
 
         return response["fullyQualifiedName"]
 
-    def create_or_update_database(self, name: str, service_fqn: str):
+    def create_or_update_database(self, metadata: CatalogueMetadata, service_fqn: str):
         """
         Define a database.
         There should be one database per data platform catalogue.
         """
         create_db = CreateDatabaseRequest(
-            name=name,
+            name=metadata.name,
+            description=metadata.description,
+            tags=self._generate_tags(metadata.tags),
             service=service_fqn,
         )
         return self._create_or_update_entity(create_db)
 
-    def create_or_update_schema(self, name: str, database_fqn: str):
+    def create_or_update_schema(self, metadata: DataProductMetadata, database_fqn: str):
         """
         Define a database schema.
         There should be one schema per data product.
         """
-        create_schema = CreateDatabaseSchemaRequest(name=name, database=database_fqn)
+        create_schema = CreateDatabaseSchemaRequest(
```

Comment:
You are collecting extra information  for schema like DPIA required but can see any custom properties created for those ?

Reply:
Functionality for custom property is available for tables  but not for database

Reply:
New ticket: https://github.com/ministryofjustice/data-platform/issues/2022